### PR TITLE
*: the value of tikv-client.copr-cache.capacity-mb should >=0

### LINKED ISSUE #24118: the value of tikv-client.copr-cache.capacity-mb should >=0
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

set tikv-client.copr-cache.capacity-mb = -1.0 
this value should >=0. 0； 0.0 means close the function.

### 2. What did you expect to see? (Required)
report error in log

### 3. What did you see instead (Required)
set successfully. 

### 4. What is your TiDB version? (Required)

v5.0.0

### PR DESCRIPTION
**Pull request by Howie59**
### What problem does this PR solve?

Issue Number: close #24118
### Check List 

Tests 
- Manual test (add detailed scripts or steps below)
- Side effects

### Release note 

- No release note

### CODE REVIEW

File: store/copr/coprocessor_cache.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ import (
 	"time"
 	"unsafe"
 
+	"github.com/pingcap/tidb/util/logutil"
+
```
Comment (wjhuang2016) [inline store/copr/coprocessor_cache.go:25]: Redundant empty line

File: store/copr/coprocessor_cache.go [hunk 2]
```diff
@@ -60,6 +62,10 @@ func newCoprCache(config *config.CoprocessorCache) (*coprCache, error) {
 	if config == nil || config.CapacityMB == 0 {
 		return nil, nil
 	}
+	if config.CapacityMB < 0 {
+		logutil.BgLogger().Info("Capacity should >= 0")
+		return nil, nil
```
Comment (wjhuang2016) [inline store/copr/coprocessor_cache.go:67]: Should return error

File: store/copr/coprocessor_cache.go [hunk 3]
```diff
@@ -60,6 +61,10 @@ func newCoprCache(config *config.CoprocessorCache) (*coprCache, error) {
 	if config == nil || config.CapacityMB == 0 {
 		return nil, nil
 	}
+	if config.CapacityMB < 0 {
+		logutil.BgLogger().Info("Capacity should >= 0")
+		return nil, errors.New("Capacity should >= 0")
+	}
```
Comment (djshow832) [inline store/copr/coprocessor_cache.go:67]: The error message is not clear enough for users. How about just merge this part into the judgement below?
`if capacityInBytes == 0` -> `if capacityInBytes <= 0`.

File: store/copr/coprocessor_cache.go [hunk 4]
```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/dgraph-io/ristretto"
 	"github.com/pingcap/errors"
 	"github.com/pingcap/kvproto/pkg/coprocessor"
+
```
Comment (mmyj) [inline store/copr/coprocessor_cache.go:27]: Delete this empty line?

**Review by mmyj — CHANGES_REQUESTED** [review]

**Pull request merged**